### ISSUE TITLE
Use mocha options provided in Settings when debugging

### ISSUE THREAD

**Issue by yusuf-kami** [maty21/mocha-sidebar#26]
## Summary
We are still using the default args value for the `path` to the tests when we run tests in debug mode using the sidebar.

## Origin
We are hardcoding the arguments we use when we run Mocha in debug mode as can be seen in the snippet:
```
~/.vscode/extensions/maty.vscode-mocha-sidebar-0.13.2/provider-extensions/runDebug.js

62|  callFunctionOnTerminate = functionOnTerminate.bind(_provider);
63|  mochaTest.args = ["./test/**/*.js"]
```
This above assignment of `args` is repeated for the methods `debugLevel` and `debugItem` as well.

## Suggestion
Parse and use the options provided in the User/Workspace Settings file as the value for the argument to be used to run Mocha in debug mode.

**Comment by maty21** [comment]
Oops I'll fix it thanks

**Comment by maty21** [comment]
fixed for version 0.13.4 commit number [22347fe2087dc4fb95b7d15f457d2e9bec189373](https://github.com/maty21/mocha-sidebar/commit/22347fe2087dc4fb95b7d15f457d2e9bec189373)

@yusuf-kami  if you can verify that it working for you it will be great thanks

**Comment by yusuf-kami** [comment]
@maty21 running in debug mode now uses the properties from the Settings file. 
Thanks for the update.
